tarantool_test.go

```diff
@@ -984,7 +1011,18 @@ func TestSQL(t *testing.T) {
 		assert.NoError(t, err, "Failed to Execute, Query number: %d", i)
 		assert.NotNil(t, resp, "Response is nil after Execute\nQuery number: %d", i)
 		for j := range resp.Data {
-			assert.Equal(t, resp.Data[j], test.Resp.Data[j], "Response data is wrong")
```

Comment:
Why this change was introduced? May this commit make existing user code invalid?

Reply:
Outdated. But everything will work as before until an user enables `go_tarantool_msgpack_v5` build flag.